lib/components/viewers/stop-viewer.js

```diff
@@ -459,9 +469,9 @@ function getFormattedStopTime (stopTime, homeTimezone, soonText = 'Due', timeFor
             style={{ color: '#888', fontSize: '0.8em', marginRight: 2 }} />
         }
       </div>
-      <div style={{ marginLeft: 20, fontSize: differentDay ? 12 : 14 }}>
-        {differentDay &&
-          <div style={{ marginBottom: -4 }}>{serviceDay.format('dddd')}</div>
+      <div style={{ marginLeft: 20, fontSize: showDayOfWeek ? 12 : 14 }}>
+        {showDayOfWeek &&
+          <div style={{ marginBottom: -4 }}>{arrivalDay.format('dddd')}</div>
```

Comment:
Please make this format string configurable.

Reply:
I'd like to avoid making things configurable for the sake of being configurable. Is there some other format that we expect users of otp-rr might want here? I suppose something like `MM/dd` might be a desirable option, but I'm not totally convinced this is necessary.

Reply:
All of the other calls in otp-rr to `moment.format` don't use hardcoded strings. When these strings are introduced it can easily lead ot inconsistencies with date formatting, so I think it would be good to have consistent formats that can be referred to so it is easy to be reason about why a time is being formatted this way.